src/components/settings-about.tsx

```diff
@@ -0,0 +1,132 @@
+  const insets = useSafeAreaInsets();
+
+  return (
+    <ScrollView
+      style={{paddingBottom: insets.bottom}}
+      contentContainerStyle={styles.content}>
+      <Spacer style={styles.animation}>
+        <LottieWrap
+          style={styles.imageStyle}
+          source={require('../../assets/animations/first-screen-animation.json')}
+          autoPlay
+          loop
+        />
+      </Spacer>
+      <Text t14 i18n={I18N.settingsAboutTitle} style={styles.title} />
+      <View style={styles.buttons}>
+        {/* <IconButton onPress={onPressRate} style={page.button}>
+          <StarIcon color={GRAPHIC_BASE_1} />
+          <Text t11 style={page.buttonText}>
+            Rate ISLM wallet App
+          </Text>
+          <Spacer />
+          <ArrowForwardIcon color={GRAPHIC_SECOND_3} />
+        </IconButton> */}
+        <IconButton onPress={onPressSite} style={styles.button}>
+          <GlobalIcon color={getColor(Color.graphicBase1)} />
+          <Text t11 i18n={I18N.settingsAboutVisit} style={styles.buttonText} />
+          <Spacer />
+          <ArrowForwardIcon color={getColor(Color.graphicSecond3)} />
+        </IconButton>
+      </View>
+      <Text t14 i18n={I18N.settingsAboutDocuments} style={styles.title} />
+      <View style={styles.buttons}>
+        <IconButton onPress={onPressDoc} style={styles.button}>
+          <DocIcon color={getColor(Color.graphicBase1)} />
+          <Text t11 i18n={I18N.settingsAboutTerms} style={styles.buttonText} />
+          <Spacer />
+          <ArrowForwardIcon color={getColor(Color.graphicSecond3)} />
+        </IconButton>
+      </View>
+      <Text t14 i18n={I18N.settingsAboutSocials} style={styles.title} />
+      <View style={styles.buttons}>
+        <IconButton onPress={onPressDiscord} style={styles.button}>
+          <DiscordIcon color={getColor(Color.graphicBase1)} />
+          <Text
+            t11
+            i18n={I18N.settingsAboutDiscord}
+            style={styles.buttonText}
+          />
+          <Spacer />
+          <ArrowForwardIcon color={getColor(Color.graphicSecond3)} />
+        </IconButton>
+      </View>
+      <Text
+        t11
+        color={getColor(Color.textBase1)}
+        i18n={I18N.settingsAboutRights}
+        style={styles.terms}
+      />
+    </ScrollView>
+  );
+};
+
+const styles = createTheme({
+  content: {
+    marginHorizontal: 20,
+    flexGrow: 1,
+  },
+  title: {
+    marginBottom: 8,
+    marginHorizontal: 4,
+    color: Color.textBase2,
+  },
+  buttons: {
+    backgroundColor: Color.bg3,
+    borderRadius: 16,
+    marginBottom: 24,
+  },
+  button: {
+    paddingHorizontal: 20,
+    paddingVertical: 12,
+    flexDirection: 'row',
+  },
+  buttonText: {
+    marginLeft: 12,
+    color: Color.textBase1,
+  },
+  terms: {
+    marginBottom: 10,
+  },
+  animation: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  imageStyle: {
+    width: windowWidth,
+    height: windowWidth * 0.9,
```

Comment:
Rewrite to `useWindowDimensions()`, because when changing the portrait mode to another, the `windowWidth` value from `'@app/helper'` will not be updated

Reply:
![image](https://user-images.githubusercontent.com/70755860/205265287-101f8829-7df4-42de-80a6-46927f2487a0.png)
normal?

Reply:
Yes!